native-script.rb

```diff
@@ -0,0 +1,55 @@
+# coding: utf-8
+
+# Only the user can manually download and install Xcode from AppStore 
+puts "Installing Xcode. Please click on 'Get Xcode' on the following dialog and install Xcode from AppStore."
+`xcode-select --install`
+
+until `pkgutil --pkg-info=com.apple.pkg.Xcode`.include? "version" do
+  puts "Waiting for Xcode do finish installing..."
+  sleep(20)
+end
+
+puts "You need to agree to Xcode license to be able to use the compilers... (You might need to provide your password.)"
+`sudo gcc`
+
+# Install all other dependencies
+puts "Installing Homebrew... (You might need to provide your password.)"
+`ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
+
+puts "Installing Homebrew-Cask... (You might need to provide your password.)"
+`brew install caskroom/cask/brew-cask`
+
+puts "Installing Java JDK... (You might need to provide your password.)"
```

Comment:
Java JDK is redundant. You can go with "Installing the Java Development Kit (JDK)"